create_ipac_tables/README

```diff
@@ -0,0 +1,15 @@
+# koa_tap_table_to_ipac_table.py
```

Comment:
This folder looks like some test/validation code i think.  I'm wondering if we should organize this into a general test folder.  This being the first test and hopefully we will add more later on.

Reply:
Get out of my head. I was thinking the same thing. I will do some refactoring to make this more test-like.